src/lib/y2network/network_manager/config_writer.rb

```diff
@@ -35,7 +35,11 @@ class ConfigWriter < Y2Network::ConfigWriter
       def write_connections(config, _old_config)
         writer = Y2Network::NetworkManager::ConnectionConfigWriter.new
         config.connections.each do |conn|
-          writer.write(conn, nil, routes_for(conn, config.routing.routes)) # FIXME
+          opts = {
+            routes: routes_for(conn, config.routing.routes),
+            parent: conn.find_master(config.connections)
+          }.reject { |_k, v| v.nil? }
+          writer.write(conn, nil, **opts) # FIXME
```

Comment:
What this "FIXME" means?

Reply:
It comes from https://github.com/yast/yast-network/commit/62fac510bb7a651dc685f78afc3762d7460fbb7c#diff-183cc708348de6ae3fa9bf93781e5524decb751c307eae9889d04abfc720350eR38 so I guess we can get rid of it already

Reply:
I will remove it in next PR